server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -113,7 +113,7 @@ public TransportBulkAction(ThreadPool threadPool, TransportService transportServ
                                ClusterService clusterService, IngestService ingestService,
                                NodeClient client, ActionFilters actionFilters, IndexNameExpressionResolver indexNameExpressionResolver,
                                AutoCreateIndex autoCreateIndex, LongSupplier relativeTimeProvider) {
-        super(BulkAction.NAME, transportService, BulkRequest::new, actionFilters, ThreadPool.Names.WRITE);
+        super(BulkAction.NAME, transportService, actionFilters, BulkRequest::new, ThreadPool.Names.WRITE);
```

Comment:
with this change in here, can we remove the unused constructor in HandledTransportAction (line 71). I believe this was the only usage

```
protected HandledTransportAction(String actionName, TransportService transportService, Supplier<Request> request, ActionFilters actionFilters, String executor)
```